tests/functional/test_check.py

```diff
@@ -0,0 +1,52 @@
+def test_check_clean(script):
+    """On a clean environment, check shouldn't return anything.
+
+    """
+    result = script.pip('check')
+    assert result.stdout == ""
```

Comment:
As a general UI principle, it's probably better if it explicitly says something like "No problems found :+1:" instead of just returning with an empty stdout.